notebooks/oggm-edu/getting_started_with_notebooks.ipynb

```diff
@@ -361,5 +361,5 @@
   }
  },
  "nbformat": 4,
- "nbformat_minor": 1
+ "nbformat_minor": 2
```

Comment:
Next time, try to commit only the files which have notable content change (I know that notebooks are messy to handle with Github...)